orchagent/natorch.cpp

```diff
@@ -115,6 +115,7 @@ NatOrch::NatOrch(DBConnector *appDb, DBConnector *stateDb, vector<table_name_wit
     if (status != SAI_STATUS_SUCCESS)
     {
         SWSS_LOG_NOTICE("Failed to get the SNAT available entry count, rv:%d", status);
+        handleSaiGetStatus(SAI_API_SWITCH, status);
```

Comment:
>handleSaiGetStatus [](http://example.com/codeflow?start=8&length=18)

If this get api is not implemented, original code will log and continue, but new code will throw. Is it intended?

Reply:
I thought the change is mainly intended to give enough warning when the SAI API is not implemented. Yet I do agree that this is not necessary for the scenarios where it could proceed without successful execution of the get operations. Removed the handling here and a few other similar scenarios.